client/my-sites/hosting/main.js

```diff
@@ -162,22 +165,27 @@ class Hosting extends Component {
 			const WrapperComponent = isDisabled || isTransferring ? FeatureExample : Fragment;
 
 			return (
-				<WrapperComponent>
-					<Layout className="hosting__layout">
-						<Column type="main" className="hosting__main-layout-col">
-							<SFTPCard disabled={ isDisabled } />
-							<PhpMyAdminCard disabled={ isDisabled } />
-							<WebServerSettingsCard disabled={ isDisabled } />
-							<RestorePlanSoftwareCard disabled={ isDisabled } />
-							<MiscellaneousCard disabled={ isDisabled } />
-							<WebServerLogsCard disabled={ isDisabled } />
-						</Column>
-						<Column type="sidebar">
-							<SiteBackupCard disabled={ isDisabled } />
-							<SupportCard />
-						</Column>
-					</Layout>
-				</WrapperComponent>
+				<>
+					<QueryKeyringServices />
+					<QueryKeyringConnections />
+					<WrapperComponent>
+						<Layout className="hosting__layout">
+							<Column type="main" className="hosting__main-layout-col">
+								<SFTPCard disabled={ isDisabled } />
+								<PhpMyAdminCard disabled={ isDisabled } />
+								<GitHubCard />
```

Comment:
I think we should put the whole thing behind a feature flag check so that we're not making a network request unless the flag is set.

Something like:

```typescript
{ isEnabled( 'hosting/github-integration' ) && (
	<>
		<QueryKeyringServices />
		<QueryKeyringConnections />
		<GitHubCard />
	</>
) }
```

Or maybe moving the `<Query*>` components into the card component since it's the card that wants the data 🤔

Reply:
I will move the queries into `<GitHubCard/>`

Reply:
Scratch that! It's too message to have it in one file. I just wrapped the queries in a conditional